package.json

```diff
@@ -42,6 +42,8 @@
     "nyc": "^13.2.0",
     "prettier": "^1.13.7",
     "pretty-quick": "^1.6.0",
+    "react": "^16.10.2",
```

Comment:
I don't think these 2 are needed here.

Reply:
I've cloned the repo, installed dependencies and tests were working fine without those two.

Reply:
Interesting on my mac tested failed without `react` and `react-dom` 😕 
I will try again, first uninstall and if I still observe it, I will share logs and a screenshot, else I will undo it.

BTW. you are also sure, that you don't have `react` nor `react-dom` installed globally?

Reply:
Okay, I revert above and tried running tests locally, which throws `Error: Cannot find module 'react'` and broke test execution for me, by:

1. `npm uninstall --save-dev react react-dom`
2 `npm run test`

<img width="1239" alt="Screenshot 2019-10-16 at 21 23 49" src="https://user-images.githubusercontent.com/914443/66951806-807ee200-f05b-11e9-89c8-fd25275ac6b4.png">

```sh
npm run test

> reaptcha-monorepo@ test /Users/axawinterthur/dev/reaptcha
> ava --verbose 'lib/**/*.test.js'


  Uncaught exception in lib/index.test.js

  /Users/axawinterthur/dev/reaptcha/lib/index.test.js:3

   2: import test from 'ava';
   3: import sinon from 'sinon';
   4: import jsdom from 'jsdom-global';

  Error: Cannot find module 'react'

  Object.require (lib/index.test.js:3:37)
  Module._compile (node_modules/pirates/lib/index.js:99:24)
  Object.newLoader [as .js] (node_modules/pirates/lib/index.js:104:7)

  ✖ lib/index.test.js exited with a non-zero exit code: 1
  ✔ utils › isAnyScriptPresent › should return true
  ✔ utils › isAnyScriptPresent › should return false
  ✔ utils › injectScript › should inject script
  ✔ utils › injectScript › should not throw when document head not present

  4 tests passed
  1 uncaught exception

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! reaptcha-monorepo@ test: `ava --verbose 'lib/**/*.test.js'`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the reaptcha-monorepo@ test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/axawinterthur/.npm/_logs/2019-10-16T19_23_21_536Z-debug.log
```

Reply:
You shouldn't use `npm` on a `yarn`-based project. There's a `yarn.lock` file which defines what should be installed. I think that `react` and `react-dom` are just dependencies of react testing tools, so you don't need to install it manually.

Btw. by doing `npm uninstall react react-dom` you uninstall it from `package.json` AND uninstall all existing versions from `node_modules`. Try deleting `node_modules`, do an `yarn install` and run the tests. I bet it works!

Reply:
Alright. Thanks  for clarification. I will do that. Just give me sec.

Reply:
Thanks, you are right 👍 
If I understand it correctly, it seems that `yarn` deals with unmet peer dependencies automatically, in contrast to `npm` 🤔 
```sh
warning " > enzyme-adapter-react-16@1.11.2" has unmet peer dependency "react@^16.0.0-0".
warning " > enzyme-adapter-react-16@1.11.2" has unmet peer dependency "react-dom@^16.0.0-0".
warning "enzyme-adapter-react-16 > enzyme-adapter-utils@1.10.1" has unmet peer dependency "react@0.13.x || 0.14.x || ^15.0.0-0 || ^16.0.0-0".
warning "enzyme-adapter-react-16 > react-test-renderer@16.4.2" has unmet peer dependency "react@^16.0.0".
warning " > react-hot-loader@4.3.5" has unmet peer dependency "react@^15.0.0 || ^16.0.0".
warning " > reaptcha@1.6.0" has unmet peer dependency "react@^16.6.3".
```

Reply:
I reverted that commit 161419affb123b97401e255d78aeeaee49b088db